2024/articles/omp-santa.pod

```diff
@@ -0,0 +1,339 @@
+Author: oodler@cpan.org
+Title: That Time Perl+OpenMP Saved Christmas 
+Topic: OpenMP 
+
+=pod
+
+=encoding utf8
+
+=for :html
+<img src="magi-camel-gifts-perl.jpg">
+
+It was Christmas Eve, and Santa was facing a nightmare. The sleigh’s new
+GPS system, upgraded for the first time in centuries from following the Star
+of Bethlehem, was malfunctioning. As Santa checked the screen, the map was a
+chaotic mess — locations were wrong, some coordinates didn’t make sense, and
+the sleigh was heading straight into the mountains instead of over the City
+of New Orleans!
+
+Santa's current position read-out indicated he was over the Appalachians at
+the following coordinates:
+
+  Sleigh Latitude  Sleigh Longitude  Sleigh Altitude (ft)
+       38.0000° N        -81.500° W              300
+
+When he should be nearly exactly at,
```

Comment:
maybe drop the comma?

Reply:
meh, I like it better than a colon but feel like something should be there